test/stringify.js

```diff
@@ -37,7 +37,8 @@ test('should not encode undefined values', t => {
 });
 
 test('should encode null values as just a key', t => {
-	t.deepEqual(fn.stringify({xyz: null, abc: null, foo: 'baz'}), 'abc&foo=baz&xyz');
+	// eslint-disable-next-line quote-props
+	t.deepEqual(fn.stringify({'x y z': null, abc: null, foo: 'baz'}), 'abc&foo=baz&x%20y%20z');
```

Comment:
Doesn't it work if you quote them all instead of only that single one? Not sure though, just guessing :).